packages/contracts/contracts/gatekeepers/zupass/ZupassGatekeeper.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
```

Comment:
Might be wrong but I don't think any of these are needed, just add the import to https://github.com/privacy-scaling-explorations/maci-platform/blob/main/packages/contracts/contracts/mocks/Mocker.sol and run the scripts with the zupass gatekeeper